src/components/UserFilterDropdown/Styles.ts

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+export const Ulist = styled.ul`
+  list-style: none;
+  padding: 0px;
+  border: solid 1px #f2f2f2;
```

Comment:
Let's not hardcode any color.
Please use this colors object:
<img width="345" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/6906e81e-f12e-4930-acb6-9c0b2133e45a">